### ISSUE TITLE
Issues thread Testing Phase 2

### ISSUE THREAD

**Issue by damienen** [Itheum/data-dex#219]

**Comment by BogdanCostinas** [comment]
Bug / Internal

Description: Mint a Data NFT and list it. In the listing process manually adjusting the sale price to "1111" and after I clicked to list I've been restricted to 1000 ( automatically change the price box to 1000 )

Step1: Go to Mint a Data NFT
Step2: Try to place it on the marketplace with more than 1000 ITHEUM tokens
Step3: The price box will be automatically adjusted top 1000 after you click on "List 1 for 1000+..."

Expected Behavior: When I add 1300 in price text box I expect to list the Data NFT for 1300, not to receive a pop up and automatically adjusted the list price to 1000

Device: Laptop
Browser: Chrome
Wallet: DeFI Wallet

![image](https://user-images.githubusercontent.com/118804443/217328345-7424b488-8502-417e-9393-ffbbfe1f7811.png)

**Comment by BogdanCostinas** [comment]
Suggestion           Internal

Description: When you go to Data Marketplace you can see a price for a listed Data NFT (e.g. 100 ITHEUM ). After you click on Procure there is a pop-up with an additional 2%. As a consumer, if you see a dataset and you want to procure it, you are misled by the initial price.
For more context - when you go to the store you want to buy something for 100 USD ( Shelf price ) and when you check out you have to pay 102 USD - you are not prepared for this and might be annoying

Steps to reproduce the issue:
1) Go to Data Marketplace
2) Pick a Data NFT
3) Click on Procure ( Check the price ) 
4) Get the additional information in the pop up message 

Expected behavior: When I want to procure a Data NFT I expect to see the full price I have to pay to acquire that particular Data NFT

![image](https://user-images.githubusercontent.com/118804443/217483091-ba875b68-7967-4b8d-a5cb-82954f03d043.png)

**Comment by BogdanCostinas** [comment]
Bug                Community

**Description**: Claims history is not visible, you cannot access that. Instead of getting the claims history you will get an error message " ER4 Could not get your recent transactions from MultiversX Blockchain


**Steps to reproduce**
1) Go to Claims ( Add some claims by requesting from faucet )
2) Claim Allocation
3) Claim Rewards
4) Click on view Claims History
5) Get the bug

Expected behavior: When I click on View Claims History I expected to see a full list of my previously claims, but instead of this I receive an error "Could not get your recent transactions from MultiversX blockchain"


Device: Laptop
Wallet: DeFi wallet
Browswer: Chrome

![image](https://user-images.githubusercontent.com/118804443/217485275-e302ef5e-4a0a-464e-b9d0-34921c329896.png)

**Comment by BogdanCostinas** [comment]
Bug              Community

**Description** : Going to burn some NFTs to increase the value of the remaining ones it's a cool feature and interesting idea, however you should not be eligible to add negative numbers in the text box

**Steps to reproduce**
1) Mint a Data NFT
2) Click on Burn
3) In the textbox you can add negative numbers( There is an error displayed but it should be restricted- no negative numbers, or no more than the actual supply owned )

**Expected Behavior:** When I select negative numbers I should be restricted from the first moment. Instead of allowing me to add negative numbers, to stick with 0 and still pop the error that you are not allowed to go below 0 

![image](https://user-images.githubusercontent.com/118804443/217488036-63812bb5-26a4-4c6f-a068-ef60607e9742.png)

**Comment by BogdanCostinas** [comment]
Bug

**Description**:While waiting for a transaction to go through, you're not able to see what's in your Data NFT Wallet/ or on Data NFT Marketplace ( Pics attached ) 

**Steps**:
1) Claim from the Claims Portal ( Sign the transaction )
2) Go to Data NFT's Menu
3) Go to Wallet/Marketplace 
4) The error is there ( You cannot view anything )
4.1 This error appears for any type of transaction that's ongoing( You can procure 1 Data NFT and then change menus - from marketplace to Wallet and vice-versa ) 

**Expected behavior:** While waiting for a transaction message to disappear from the interface I expect to be able to see what's on the Marketplace or in my Wallet

![Pic1 No data](https://user-images.githubusercontent.com/118804443/217495900-79409610-7e74-4d48-9928-0f4711feccba.JPG)
![image](https://user-images.githubusercontent.com/118804443/217495949-c333fd39-a7ff-4001-952f-2173c8f95365.png)

**Comment by BogdanCostinas** [comment]
Bug/Suggestion   Community/Internal

**Description:** There is a timer between 2 data nft's minting process. This timer is displayed nowhere only after you try to mint your second Data NFT you get an error which will tell you when you'll be capable of minting another Data NFT

**Steps to reproduce:**
1) Mint a Data NFT
2) Right away, go again and mint another Data NFT
3) Get the error

**Expected Behavior:**
1. I expect to be notified if there is a limit to how many different datasets I can mint in a time interval
2. I expect no restrictions in terms of how many different datasets I am minting in a given interval 
3. If 1 is met, then a timer should be added
![image](https://user-images.githubusercontent.com/118804443/217499254-def9bdf5-362c-4a87-8a9f-a24a6b5c1af0.png)

**Comment by BogdanCostinas** [comment]
**Bug**                  ( Will try to look for another pattern as well )

**Description**: Trying to burn some newly created NFTs will give you an error. This is the pattern that found by me 
If I mint XYZ Data NFTs and try to burn more than X (X+1) I will get the error ( 800 NFTs, I'll receive the error when trying to burn 9, for 500 NFTs I got the error when trying to burn 6  )

Steps:
1. Go to advertise data
2. Create a Data NFT ( Choose more copies )
3. Go to Burn
4. Increase the burning amount by 1, multiple times,  in the text box until you get the error

Extra content ( if you mint 300 NFT's you will get the error when trying to burn 4 )


Expected behavior: To be able to burn as many NFTs i want as long as the amount to be burned is less than the owned supply
![image](https://user-images.githubusercontent.com/118804443/217516989-f71fd335-78b9-4dec-aa9f-ba572436c77e.png)

**Comment by BogdanCostinas** [comment]
Bug

**Description**: Strange behavior of My listed data NFTs after checking the last page of listed Data NFTs on the public marketplace - The description disappear ( The image disappear from time to time ) 

**Steps**:
1 Go to the public marketplace
2 Push the right arrow to see the last page
3 Go to My listed Data NFT and get the error

**Expected Behavior:** After doing all the steps I expect to be capable of viewing my listed data NFT's
https://media.discordapp.net/attachments/991963100674785361/1072575828564643851/image.png?width=1040&height=585
https://media.discordapp.net/attachments/991963100674785361/1072575828866637824/image.png?width=1040&height=585
https://media.discordapp.net/attachments/991963100674785361/1072575829109915728/image.png?width=1040&height=585

**Comment by BogdanCostinas** [comment]
Bug

**Description:** When trying to burn NFTs you can encounter another issue. This will be explained with an example" I have 4 NFTs and I want to burn 5 - it's impossible, but if I want to burn 3000000000000000000 I am able to do that in the textbox )

**Steps to reproduce:**
1. Mint multiple copies of a dataset ( eg. 3 )
2. Go to Burn
3. Using the keyboard, make sure that the first number in the textbox is smaller than the first character in the number of the copies; then continue to add numbers
e.g If we minted 3 NFTs we can burn "22333333 or 14444444 or 24444444....) 

**Expected behavior:** When trying to burn more Data NFTs to be restricted between 1 and the number of copies I own
![image](https://user-images.githubusercontent.com/118804443/217539690-efc26d60-af98-4bd2-82bc-511a017cce6c.png)

**Comment by BogdanCostinas** [comment]
Bug

Description: The interface is not refreshing while procuring 1 data nft. While the transactions is ongoing you'll be able to see the error

Steps
1. Go to Public Marketplace
2. Procure 1 data nft
3. Go to my listed data nft's while the transaction is in progress
4. Get the error, the interface is not refreshed, you'll see all of the listed Data NFTs on the public marketplace as yours

Expected behavior: Switching pages between My listed data nfts and public marketplace while a transaction is in progress should not affect the information displayed on Public Marketplace and My Listed Data NFTs


![image](https://user-images.githubusercontent.com/118804443/217541189-9f2c3099-c7eb-4554-9e1d-6bd5953a90b3.png)
![image](https://user-images.githubusercontent.com/118804443/217541246-f05497de-6b4f-46c3-acc6-f31b82073ef3.png)

**Comment by BogdanCostinas** [comment]
**Suggestion from Community**

The data description text field should be allowed to take more than 250 characters. There's a lot to write to describe data. The description is more like a selling point that would attract buyers to your product.

**Comment by BogdanCostinas** [comment]
Bug

**Description**: You can update the price for a specific listed Data NFT, however, the "price" should not be allowed to go below 0. This occurs when you use the up and down arrow to modify the price, using the down arrow you can get negative numbers

**Steps to reproduce:**
1. List a Data NFT on Public Marketplace
2. Go to my listed Data NFTs
3. Update price
4. Use the Down arrow and go below 0.

**Expected behavior:** The negative numbers should not be allowed in the text box
![image](https://user-images.githubusercontent.com/118804443/217547391-15b3c144-15f4-4ea1-97e5-dbc6a0a0aac9.png)

**Comment by BogdanCostinas** [comment]
Community Suggestion 

If you are not the CREATOR of a Data NFT you should not be able to burn the NFT.
e.g. What If I purchase all the NFTs from somebody, burn them all and mint another data nft to benefit more? ( Trade as it is mine )
